src/en/developer-debugging.md

```diff
@@ -148,15 +148,15 @@ terminating.
 
 Prior to version 2.0, hooks returning an error will block until the user
 takes an action to retry them manually, by issuing the command `juju resolved
---retry unit-name/#` for the affected unit. After issuing the retry command,
-go back to the debug-hooks session to interact with the Juju environment.
+unit-name/#` for the affected unit. After issuing the retry command, go back
```

Comment:
I think this is clearer if 'After issuing the retry command' is 'After issuing the resolved command'.